source/GameData.cpp

```diff
@@ -946,6 +958,19 @@ void GameData::LoadFile(const string &path, bool debugMode)
 			for(const DataNode &child : node)
 				list.push_back(child.Token(0));
 		}
+		else if(key == "categories" && node.Size() >= 2)
```

Comment:
Move the token value check up here, so that if an unsupported category is passed, a warning is generated:
`else if(key == "categories" && node.Size() >= 2 && (node.Token(1) == "outfit" || node.Token(1) == "ship"))`